shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "List of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-ref-schema.json"
+            }
+        },
+        "options": {
```

Comment:
I'm not convinced the `options` is the best name to describe this property.
Perhaps `outputs` or `resultDetails` or a different name that better fits this property?

Reply:
The purpose is to hold data produced by the workflow. So the `outputs` sounds best here.